code/modules/clothing/suits/ego_gear/_ego_gear.dm

```diff
@@ -3,6 +3,7 @@
 	desc = "You aren't meant to see this."
 	icon = 'icons/obj/clothing/ego_gear/suits.dmi'
 	worn_icon = 'icons/mob/clothing/ego_gear/suit.dmi'
+	flags_inv = HIDEJUMPSUIT
```

Comment:
You would have to check with Redacted if Remorse is supposed to go above the suit or not.

Reply:
They did mention that it'd be nice if it hid jumpsuit.